packages/astro/src/runtime/client/dev-toolbar/ui-library/badge.ts

```diff
@@ -15,9 +15,11 @@ export class DevToolbarBadge extends HTMLElement {
 
 		if (this.hasAttribute('badge-style'))
 			this.badgeStyle = this.getAttribute('badge-style') as BadgeStyle;
+	}
 
+	connectedCallback() {
 		const classes = [`badge--${this.size}`, `badge--${this.badgeStyle}`];
-		this.shadowRoot.innerHTML = `
+		this.shadowRoot.innerHTML += `
```

Comment:
This change will be removed by https://github.com/withastro/astro/pull/10186, but it has the same effect